styles/style.css

```diff
@@ -239,13 +241,15 @@ div#content_block { width: 70% }
 /* Pinterest Button */
 .rtsocial-pinterest-button,
 #rtsocial-ord-pin label { background-position: 0 -598px; border: 1px solid #c9c5c5; border-color: #E8E4E4 #C9C5C5 #C9C5C5; border-radius: 4px 4px 4px 4px; display: block; height: 22px; text-indent: -9999px; text-transform: uppercase; width: 47px; margin: 0 auto; }
+.rtsocial-pinterest-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -598px no-repeat !important;}
```

Comment:
Use of `!important` is causing the PHPCS test failure.

Reply:
The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files